waspc/examples/todoApp/src/TestRpcTypes.ts

```diff
@@ -0,0 +1,112 @@
+import { AuthUser } from 'wasp/auth'
+import { getMe } from 'wasp/client/auth'
+import {
+  getTask,
+  getTasks,
+  createTask,
+  updateTaskIsDone,
+  deleteCompletedTasks,
+  toggleAllTasks,
+  getNumTasks,
+  getDate,
+  getAnything,
+  getTrueVoid,
+} from 'wasp/client/operations'
+import { Task } from 'wasp/entities'
+import { Payload } from 'wasp/server/_types'
+
+// For the details of this specification, see
+// https://github.com/wasp-lang/wasp/pull/1090#discussion_r1159732471
+
+// This should be [],
```

Comment:
Maybe put a TODO here

Reply:
I'll do you one better: https://github.com/wasp-lang/wasp/issues/2004